quantities/tuples.hpp

```diff
@@ -52,13 +42,9 @@ struct DerivativesGenerator;
 using internal_tuples::is_tuple;
 using internal_tuples::is_tuple_v;
 
-template<template<typename> class Transform, typename Tuple>
-using Apply = typename internal_tuples::ApplyGenerator<Transform, Tuple>::Type;
-
-template<template<typename, typename> class Transform,
-         typename LTuple, typename RTuple>
-using Apply2 =
-    typename internal_tuples::Apply2Generator<Transform, LTuple, RTuple>::Type;
+template<template<typename> class Transform, typename... Tuples>
```

Comment:
Why `class` here and `typename` above?  Also why no `...` after the innermost `typename`?

Reply:
Done.